recipes/_windows.rb

```diff
@@ -19,6 +19,22 @@
 
 windows = node["sensu"]["windows"].dup
 
+if node['kernel']['machine'] =~ /x86_64/
+  kernel = 'x64'
+else
+  kernel = 'x86'
+end
+
+if node['platform_version'].to_f == 6.1
+  node.default['sensu']['msi_repo_url'] = 'https://repositories.sensuapp.org/msi/2008r2'
```

Comment:
I think we should specify the attribute as `https://repositories.sensuapp.org/msi/` and then just have a local variable here to keep track of version/edition just like we do with kernel.